pyqg/parameterizations.py

```diff
@@ -286,3 +286,78 @@ def __call__(self, m):
 
     def __repr__(self):
         return f"ZannaBolton2020(κ={self.constant:.2e})"
+
+def Gaussian_Filter(w, m, FGR):
+    w_fft = m.fft(w)
+    Delta = m.dx * FGR
+    k2 = m.k**2 + m.l**2
+    G = np.exp(-k2*Delta**2/24)
+    return m.ifft(G * w_fft)
+
+def deconvolve(w, m, FGR, order):
+    wd = w
+    r = w
+    for j in range(order):
+        r = r - Gaussian_Filter(r, m, FGR)
+        wd = wd + r
+    return wd
+
+def SFS(q, u, v, m, FGR):
+    qf = Gaussian_Filter(q, m, FGR)
+    uf = Gaussian_Filter(u, m, FGR)
+    vf = Gaussian_Filter(v, m, FGR)
+    quf = Gaussian_Filter(q*u, m, FGR)
+    qvf = Gaussian_Filter(q*v, m, FGR)
+    SFSu = qf*uf - quf
+    SFSv = qf*vf - qvf
+    return SFSu, SFSv 
```

Comment:
These methods should probably get [numpy-style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy) as well -- e.g.:
```
def SFS(q, u, v, m, FGR):
  """Computes the subfilter stress-or-something-or-other of the quantity :code:`q` 
  based on velocities :code:`u` and :code:`v`, a model :code:`m`, and a filter to
  grid ratio :code:`FGR`.

  Parameters
  ----------
    q : array
       The quantity for which we're computing subfilter stress (?)

    ...

  Returns
  -------
    SFS : tuple of two arrays
      the thingy
  """
```